packages/testlab/README.md

```diff
@@ -287,6 +289,39 @@ describe('MyApp', () => {
 });
 ```
 
+### `createUnexpectedHttpErrorLogger`
+
+An error logger that logs the error only when the HTTP status code is not the
+expected HTTP status code. This helps surpress the console from printing an
+expected error.
```

Comment:
> This helps surpress the console from printing an expected error.

Let's improve this part. My proposal - feel free to change it as you like:

```markdown
This is useful when writing tests for error responses:

- We don't want any error message printed to the console when 
  the server responds with the expected error and the test passes.

- When something else goes wrong and the server returns an unexpected 
  error status code, we do want an error message to be printed to the console 
  so that we have enough information to troubleshoot the failing test.
```